content/_data/changelogs/weekly.yml

```diff
@@ -1785,6 +1785,34 @@
         - issue: 48754
         - url: https://groups.google.com/forum/#!topic/jenkinsci-dev/-VQ5YU0yYhg
           title: Retrospective thread in the developer mailing list
+- version: "2.101"
+  date: 2018-01-07
+  changes:
+    - type: rfe
+      pull: 3214
+      message: >
+        Log EOF exceptions on a <code>FINE</code> level instead of <code>WARNING</code> when the Jetty connection is closed by peer.
+    - type: bug
+      pull: 3178
+      issue: 48447
+      message: >
+        Fix HTTP 404 error when clicking on <em>New View</em> sidebar link from another view.
```

Comment:
This appears to have been reverted as one of the commits in your revert PR.

Reply:
Not true, https://github.com/jenkinsci/jenkins/pull/3218/commits/cc9b1053ad27b5569fd599df9fdd5bc34242f97e exists without purpose, and the squashed commit message is wrong.